### ISSUE TITLE
Environment variable in Network-Watcher of the Deployment for SERVER_PORT should not read from .Values.service.port

### ISSUE THREAD

**Issue by bob-bn** [soketi/charts#23]
Hello,

When we launch the Socketi with Network watcher enabled, we get the below error:

```
Starting the watcher...
Namespace: dev
Pod name: soketi-soketii-5f845d945b-zj5mn
Server port: 6001
Memory threshold: 85%
Monitoring interval: 1s

In ChecksCurrentPod.php line 97:
                                                                               
  App\Commands\WatchNetworkCommand::getUsage(): Return value must be of type   
  array, null returned
```

After investigation, the following file / Line [https://github.com/soketi/charts/blob/master/charts/soketi/templates/deployment.yaml#L76](deployment.yaml) is wrong and should not read the SERVER_PORT from `.Values.service.port` but instead should be the metrics server port 9601. 

My suggestion is to add in values the following  `.Values.metrics.port` and use it across the board where needed and in SERVER_PORT env of the network watcher.

I hope this helps everyone else as well!

**Comment by J0sh0nat0r** [comment]
I ran into the same issue and attempted a fix in #19, however the response I got would seem to indicate that it works for @rennokki so maybe this is something environment specific?

**Comment by rennokki** [comment]
It was environment specific, i passed my own variables that overwritten the default ones. 😬 
Fixed in chart `1.0.2`.
